test/fuzz/fuzz_doc_symbols.cc

```diff
@@ -45,18 +50,15 @@ extern "C" int LLVMFuzzerInitialize(const int *argc, const char ***argv) {
 }
 
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, const std::size_t size) {
-    auto lspWrapper = mkLSPWrapper();
+    std::string contents((const char *)data, size);
+    auto lspWrapper = mkLSPWrapper(contents);
     int nextId = 0;
     sorbet::test::initializeLSP(rootPath, rootUri, lspWrapper, nextId);
-    std::string contents((const char *)data, size);
-    auto fileUri = sorbet::test::filePathToUri(rootUri, "file.rb");
-    lspWrapper.getLSPResponsesFor(sorbet::test::LSPMessage(std::make_unique<sorbet::test::NotificationMessage>(
-        "2.0", sorbet::test::LSPMethod::TextDocumentDidOpen,
-        std::make_unique<sorbet::test::DidOpenTextDocumentParams>(
-            std::make_unique<sorbet::test::TextDocumentItem>(fileUri, "ruby", 1, contents)))));
-    lspWrapper.getLSPResponsesFor(sorbet::test::LSPMessage(std::make_unique<sorbet::test::RequestMessage>(
-        "2.0", nextId++, sorbet::test::LSPMethod::TextDocumentDocumentSymbol,
-        std::make_unique<sorbet::test::DocumentSymbolParams>(
-            std::make_unique<sorbet::test::TextDocumentIdentifier>(fileUri)))));
+    const auto responses =
+        lspWrapper.getLSPResponsesFor(sorbet::test::LSPMessage(std::make_unique<sorbet::test::RequestMessage>(
+            "2.0", nextId++, sorbet::test::LSPMethod::TextDocumentDocumentSymbol,
+            std::make_unique<sorbet::test::DocumentSymbolParams>(
+                std::make_unique<sorbet::test::TextDocumentIdentifier>(fileUri)))));
+    sorbet::fatalLogger->info("num responses: {}", responses.size());
```

Comment:
Ah, is this just to check that you received a response?

You could `ENFORCE(!responses.empty())`.

Reply:
could I actually ENFORCE that it's exactly 1?

Reply:
(yes, this is just to sanity check.)

Reply:
Hmm, I think so. We shouldn't send back any other pieces of information for the request.